nomos-services/network/src/backends/libp2p.rs

```diff
@@ -131,6 +162,17 @@ impl NetworkBackend for Libp2p {
                             }
                         };
                     }
+                    Ok(msg) = mixnet_rx.recv() => {
+                        let MixnetMessage { topic, message } = MixnetMessage::from_bytes(&msg);
+                        match swarm.broadcast(&topic, message) {
+                            Ok(id) => {
+                                tracing::debug!("broadcasted message with id: {id} tp topic: {topic}");
+                            }
+                            Err(e) => {
+                                tracing::error!("failed to broadcast message to topic: {topic} {e:?}");
+                            }
+                        }
+                    }
                 }
```

Comment:
What do you guys think about this strategy, using mixnet only for the first hop of gossipsub? This is one of strategies that we discussed. Although we haven't verify this startegy in details, I'm getting an impression that this will probably meet our privacy requirements, as Lighthouse guys [said](https://github.com/logos-co/nomos-node/issues/273#issuecomment-1657545446).